<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xAnatomy</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="SagittalWidget" name="widget_sagittal" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="CoronalWidget" name="widget_coronal" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="AxialWidget" name="widget_axial" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="VolumeWidget" name="widget_volume" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_right">
          <property name="text">
           <string>ðŸ‘‰</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_rotate_inv">
          <property name="text">
           <string>ðŸŒœ</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_left">
          <property name="text">
           <string>ðŸ‘ˆ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_down">
          <property name="text">
           <string>ðŸ‘‡</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_rotate">
          <property name="text">
           <string>ðŸŒ›</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_up">
          <property name="text">
           <string>ðŸ‘†</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_playSlice">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_reset">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>976</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SagittalWidget</class>
   <extends>QWidget</extends>
   <header>sagittalwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CoronalWidget</class>
   <extends>QWidget</extends>
   <header>coronalwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AxialWidget</class>
   <extends>QWidget</extends>
   <header>axialwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QWidget</extends>
   <header>volumewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../xAnatomy.qrc"/>
 </resources>
 <connections/>
</ui>
